BookOfHabits/BookOfHabits.sln

```diff
@@ -3,7 +3,25 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
```

Comment:
First of all, before creating PR you should always check target branch for merging. You need to switch from 'main' to 'develop' branch, because with accordance to Git flow, we work with 'develop' branch only on step of development.

Additionally, you need to check all files for merging in order to not allow appearing inappropriate files which are not directly related to your changes.

And could you please mention number of task in the name of PR? For instance 'Task 22 Add EF for Diary microservice'